src/containers/AccountConnection.jsx

```diff
@@ -120,15 +121,15 @@ class AccountConnection extends Component {
       })
   }
 
-  updateAccount (connector, account, values) {
+  updateAccount (connector, account, values, successTimeout) {
     Object.assign(account.auth, values)
 
     this.setState({ submitting: true })
 
     return this.store.updateAccount(connector, account, values)
     .then(account => {
       this.setState({ account: account })
-      return this.store.runAccount(connector, account)
+      return this.store.runAccount(connector, account, successTimeout)
```

Comment:
So it's not what we want. We want to wait for the end of the execution by default. That means all running inside collect will be timedout to 20s. What about cozy-collect AccountConnection where `state.successTimeout` is `undefined`?